Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -981,5 +981,13 @@ define([
         return destroyObject(this);
     };
 
+    /**
+     * Generates an HTML string of the statistics
+     * @param {Cesium3DTileset} tileset The tileset
+     * @param {Boolean} isPick Whether this is getting the statistics for a picked feature
```

Comment:
Looks good to me, I would just change this description to `Whether this is getting the statistics for the pick pass`